moto/elasticache/exceptions.py

```diff
@@ -16,6 +17,8 @@
 class ElastiCacheException(RESTError):
 
     code = 400
+    extended_templates = {"ecerror": EXCEPTION_RESPONSE}
```

Comment:
The `self.templates["ecerror"] = ERROR...` can now be removed in the `__init__`-method, right?

I saw the same in the `sdb`, `elasticbeanstalk` and `s3` modules.

Reply:
done